modules/bitgo/src/v2/coins/near.ts

```diff
@@ -155,10 +158,26 @@ export class Near extends BaseCoin {
    * Explain/parse transaction
    * @param params
    */
-  explainTransaction(
+  async explainTransaction(
     params: ExplainTransactionOptions,
-  ): Promise<any> {
-    throw new MethodNotImplementedError('Near recovery not implemented');
+  ): Promise<NearTransactionExplanation> {
+    const factory = accountLib.register(this.getChain(), accountLib.Near.TransactionBuilderFactory);
+    let rebuiltTransaction;
+
+    try {
+      let rawTxBase64 = params.txPrebuild.txHex;
+      if (HEX_REGEX.test(params.txPrebuild.txHex)) {
+        rawTxBase64 = Buffer.from(params.txPrebuild.txHex, 'hex').toString('base64');
+      }
+      const transactionBuilder = factory.from(rawTxBase64);
+      rebuiltTransaction = await transactionBuilder.build();
+    } catch {
+      throw new Error('Invalid transaction');
+    }
+
+    const explainedTransaction = (rebuiltTransaction as accountLib.BaseCoin.BaseTransaction).explainTransaction();
+
+    return explainedTransaction as NearTransactionExplanation;
```

Comment:
I think we can just add the type to the statement instead of use 'as'
`let rebuiltTransaction : accountLib.BaseCoin.BaseTransaction;`

and then 
    `const explainedTransaction: NearTransactionExplanation = rebuiltTransaction.explainTransaction();
return explainedTransaction;`